packages/shared/src/components/MainFeedLayout.tsx

```diff
@@ -122,11 +126,15 @@ export default function MainFeedLayout({
     hasFiltered: !alerts?.filter,
     hasUser: !!user,
   });
+  // console.log(feedName);
   const feedVersion = useFeature(feature.feedVersion);
   const searchVersion = useFeature(feature.search);
+  const onboardingV4dot5 = useFeature(feature.onboardingV4dot5);
```

Comment:
@idoshamun Prime example, this will evaluate everyone now, technically we should only enroll new users, but it's almost impossible as to on which level we need this.

Love to hear your opinion on this matter. (non urgent)

Reply:
Everything inside is a whole feed component thats also used by other things..

This is what i mentioned many cases its not just a simple new thing, but logic on existing components.

Making it two makes it almost impossible to maintain (imo)

Reply:
Well guess the main issue is the rule for evaluation here. (bit of a edge-case)
Target only people who just started onboarding.

Basically you want to say, if you didn't get this feature from onboarding don't do anything 😅

Reply:
Does the "new user" evaluation not come from GB targeting?

Reply:
It's possible to that via GB. But the ideal rule of thumb is to use the feature only when you need it. I should think maybe there are exclusion that we can relax it a bit, new user as an example